runtime/src/tests.rs

```diff
@@ -130,6 +137,83 @@ fn new_test_ext() -> sp_io::TestExternalities {
     ext
 }
 
+fn call_transfer(dest: AccountId32, value: u128) -> Box<RuntimeCall> {
+    Box::new(RuntimeCall::Balances(BalancesCall::transfer_allow_death {
+        dest: MultiAddress::Id(dest),
+        value,
+    }))
+}
+#[test]
+fn test_multisig_send_and_return() {
```

Comment:
I agree with @la10736 on this test testing the multisig pallet rather than our own runtime, so I would also be in favor of having something simpler just to check that you can compile a test that uses a multisig exstrinsic.
But if you do want to keep it, a small suggestion would be to replace hardcoded values with named variables; that might make the test more readable. For example, `500` could become something like `SEND_AMOUNT`, and `account_ids[0]` might become `depositor_id`.